main/network.c

```diff
@@ -545,7 +541,7 @@ PHPAPI int php_network_parse_network_address_with_port(const char *addr, zend_lo
 
 	/* first, try interpreting the address as a numeric address */
 
-#if HAVE_IPV6 && HAVE_INET_PTON
+#if HAVE_IPV6
```

Comment:
```suggestion
#ifdef HAVE_IPV6
```
think it should always test definition

Reply:
Agreed, I have a long-standing (very outdated PR (https://github.com/php/php-src/pull/5526) now) to try to enable `-Wundef` to check these sorts of issues at compile time.

Reply:
Do you need help with this PR, @Girgias ?

Reply:
You can take it over, I was hitting some issue with YACC generating something that I couldn't check.